packages/backend/src/dbt/translator.mock.ts

```diff
@@ -65,6 +65,30 @@ const COLUMN_WITH_DEFAULT_TIME_INTERVALS: Record<string, DbtModelColumn> = {
     },
 };
 
+const COLUMN_WITH_NO_TIME_INTERVALS: Record<string, DbtModelColumn> = {
```

Comment:
I see you added the mocks for the new scenarios but no new tests in `/lightdash/packages/backend/src/dbt/translator.test.ts`

Reply:
had a go at this